eye2bids/edf2bids.py

```diff
@@ -302,19 +348,25 @@ def edf2bids(
     )
 
     # CONVERSION events
-    asc_file = _convert_edf_to_asc(input_file)
+    events_asc_file = _convert_edf_to_asc_events(input_file)
+    samples_asc_file = _convert_edf_to_asc_samples(input_file)
 
-    events = _load_asc_file(asc_file)
-    df_ms = _load_asc_file_as_df(asc_file)
-    df_ms_reduced = _load_asc_file_as_reduced_df(asc_file)
+    events = _load_asc_file(events_asc_file)
+    df_ms = _load_asc_file_as_df(events_asc_file)
+    df_ms_reduced = _load_asc_file_as_reduced_df(events_asc_file)
 
     if metadata_file is None:
         metadata = {}
     else:
         with open(metadata_file) as f:
             metadata = yaml.load(f, Loader=SafeLoader)
 
-    # to json
+    # file naming check
+
+    filename = Path(input_file).stem
```

Comment:
changed this line to make sure that the file would be created in the correct folder